.github/workflows/check_deps_integrity.yaml

```diff
@@ -1,11 +1,12 @@
 name: Check Rebar Dependencies
 
-on: [pull_request, push]
+on:
+  pull_request:
 
 jobs:
   check_deps_integrity:
-    runs-on: ubuntu-20.04
-    container: ghcr.io/emqx/emqx-builder/5.0-28:1.13.4-25.1.2-2-ubuntu20.04
+    runs-on: ubuntu-latest
```

Comment:
Just wondering, why do we have `latest` somewhere (deps integrity) and also `22.04` somewhere (code style check) while in both cases a container is used?

Reply:
Not sure tbh.
Replaced with `-latest` in less important workflows, and bumped to 22.04 in more critical ones.